### ISSUE TITLE
unicode labels

### ISSUE THREAD

**Issue by atoomic** [rurban/perl-compiler#318]
Env 5.22.0

probably missing an encoding somewhere ( or a double encoding ? )

````
> perl -e 'use utf8; ＬＯＯＰ: { last ＬＯＯＰ }'
> perlcc -r -e 'use utf8; ＬＯＯＰ: { last ＬＯＯＰ }'
Label not found for "last ＬＯＯＰ" at -e line 1.
````

**Comment by atoomic** [comment]
same issue with redo
```
> perl -e 'use utf8; my $c = 0; ＬＯＯＰ: { print qq{ok\n}; redo ＬＯＯＰ unless $c++ }'
ok
ok
> perlcc -r -e 'use utf8; my $c = 0; ＬＯＯＰ: { print qq{ok\n}; redo ＬＯＯＰ unless $c++ }'
ok
Wide character in redo at -e line 1.
Label not found for "redo ＬＯＯＰ" at -e line 1.

**Comment by rurban** [comment]
That's an upstream B bug. label misses the utf8 flag, it uses just 
```
/* B::COP::label */
		ret = sv_2mortal(newSVpv(CopLABEL(cCOPo),0));
```
I'll fix it in C.xs

**Comment by rurban** [comment]
Fixed with C 1.52_16 and CC 1.16_01
